cmd/create_device.go

```diff
@@ -57,7 +58,7 @@ var createDeviceCmd = &cobra.Command{
 	Short: "Creates a device",
 	Long: `Example:
 
-packet device create --hostname [hostname] --plan [plan] --facility [facility_code] --operating-system [operating_system] --project-id [project_UUID]
+packet device create --hostname [hostname] --plan [plan] --metro [metro_code] --facility [facility_code] --operating-system [operating_system] --project-id [project_UUID]
```

Comment:
The Cobra docs say that `Use` should include this argument list. The prescribe a format that we haven't been using:
```
create [--hostname hostname] --plan plan { --metro metro_code | --facility facility_code } --operating-system operating_system --project-id project_id
```

Reply:
Adopting that format might help us better illustrate the mutually exclusive facility and metro arguments.